pymobiledevice3/services/dvt/instruments/process_control.py

```diff
@@ -44,7 +45,7 @@ def kill(self, pid: int):
         self._channel.killPid_(MessageAux().append_obj(pid), expects_reply=False)
 
     def launch(self, bundle_id: str, arguments=None, kill_existing: bool = True, start_suspended: bool = False,
-               environment: typing.Mapping = None) -> int:
+               environment: typing.Mapping = None, extra_options: dict = {}) -> int:
```

Comment:
empty dicts arguments in python should be initialized to None and only then checked since they are compiled as statically objects

https://note.nkmk.me/en/python-argument-default/